include/onnxruntime/core/graph/graph.h

```diff
@@ -407,6 +407,12 @@ class Node {
   bool ClearAttribute(const std::string& attr_name);
 #endif  // !defined(ORT_MINIMAL_BUILD) || defined(ORT_EXTENDED_MINIMAL_BUILD)
 
+  /**
+  * Clears removable attributes. These are no longer needed after the initialization
+  * of the session.
+  */
+  int PruneRemovableAttributes(const InlinedVector<std::string>& removable_attributes);
```

Comment:
> InlinedVector<std::string>&  [](http://example.com/codeflow?start=36&length=29)

Per coding standards this should be gsl::span<const std::string>

Reply:
done